assets/js/base/components/cart-checkout/shipping-rates-control-package/index.tsx

```diff
@@ -34,21 +35,28 @@ interface Destination {
 	country: string;
 }
 
+export interface PackageData {
+	destination: Destination;
+	name: string;
+	// eslint-disable-next-line camelcase
+	shipping_rates: CartShippingRateItemShippingRate[];
```

Comment:
This wasn't changed in this PR, but do you think we should translate shipping_rates to shippingRates after receiving data from the API or leave in snake case?

Reply:
I think we should - but not in this PR, I'm not sure how long it'll take to find all the places this happens in, and this PR is pretty big already. So, I'll make a new issue for it to be done next cooldown.